README.md

```diff
@@ -1 +1,69 @@
-# restli-csharp
\ No newline at end of file
+# Rest.li C#
+
+## How to Build
+
+- clone the repository locally and move into directory
+    - `git clone https://github.com/mchen07/restli-csharp.git`
+    - `cd restli-csharp`
+- run the gradle script to generate data templates:
+    - on Windows: `gradlew.bat clean build`
+    - on Mac/Linux: `./gradlew clean build`
+- open `restlicsharp.sln` in Visual Studio:
+    - build the solution
+    - run all tests for the solution to ensure everything is setup
```

Comment:
This reminds me of one thing, all your tests are hardcoding your rest.li server endpoint, this needs to be documented. Maybe mention that your integration test relies on an assumption of running a rest.li server somewhere.